test/test_intensify/test_successive_halving.py

```diff
@@ -62,14 +62,16 @@ def test_init_1(self):
             instances=[1, 2, 3], n_seeds=2, initial_budget=None, max_budget=None, eta=2)
 
         self.assertEqual(len(intensifier.inst_seed_pairs), 6)  # since instance-seed pairs
+        self.assertEqual(len(intensifier.instances), 3)
         self.assertEqual(intensifier.initial_budget, 1)
         self.assertEqual(intensifier.max_budget, 6)
-        self.assertEqual(intensifier.num_initial_challengers, 4)  # 2 iterations
+        self.assertListEqual(intensifier.n_configs_in_stage.tolist(), [4.0, 2.0, 1.0])
```

Comment:
Could you please change the original attribute in intensifier to be a list (unless there's a reason not to do so).

Reply:
It is a numpy array and it could not be compared directly. So I had to switch it to a list here.